### DISCUSSION TITLE
ionization cross section

### DISCUSSION THREAD

**Discussion by SillyTom** [SmileiPIC/Smilei#784] [Q&A]
Dear developer, how is the normalization coefficient derived in the code?
CollisionalIonization.cpp     Line 77 :
double normalization = 2.81794e-15 * reference_angular_frequency_SI / ( 2.*299792458. ); // r_e omega / 2c

in the reference (Pérez et al., Phys. Plasmas 19, 083104 (2012)), the coefficient seems to be  2*pi*r_e*r_e

By the way, could you please explain how the cross section matches the simulation geometry (1D,2D,3D)?

**Comment by mccoys** [comment]
The cross section is later multiplied by other factors to convert it to a probability. It essentially ends up being multiplied by a time and by a density. You can look up the normalized units of these quantities to realize that the normalized cross-section should have the coefficient you mentioned in order to follow the units in the code.

I don't understand your last question. Please explain a bit more what you mean.
